client/landing/jetpack-cloud/lib/contact-support-url.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { JETPACK_CONTACT_SUPPORT } from 'lib/url/support';
+
+/**
+ * Creates a URL that refers to the Jetpack 'Contact Support' page,
+ * with accompanying useful information about the reason for the
+ * support request.
+ *
+ * @param {string} [siteUrl]	A site URL
+ * @param {string} [scanState]	The current state of Jetpack Scan/Backup
+ * @returns {string} 			The support request URL
+ */
+export default function contactSupportUrl( siteUrl, scanState ) {
+	// NOTE: This is an extremely simple, early implementation;
+	// if we add more options in the future, we'll want
+	// to come back and change how we build this query string
+	const params = [];
+	if ( siteUrl ) {
+		params.push( `url=${ siteUrl }` );
+	}
+
+	if ( scanState ) {
+		params.push( `scan-state=${ scanState }` );
+	}
+
+	// At present, JETPACK_CONTACT_SUPPORT already contains an initial query string
+	// parameter, so we can safely append all our parameters using '&'
+	return [ JETPACK_CONTACT_SUPPORT, ...params ].join( '&' );
```

Comment:
Its great that we are using JETPACK_CONTACT_SUPPORT constant here but can we also use 
addQueryArgs() instead of .join( '&') ? This will make the url harder to break.

Reply:
I'm glad we have something like that -- thanks for pointing it out! I've made changes so that we're using `addQueryArgs` now.